ios/GoldWallet/AppDelegate.m

```diff
@@ -26,7 +26,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge
                                                       moduleName:[ReactNativeConfig envFor:@"APPLICATION_NAME"]
                                                initialProperties:nil];
-  rootView.backgroundColor = [UIColor whiteColor];
+  if (@available(iOS 13.0, *)) {
+      rootView.backgroundColor = [UIColor systemBackgroundColor];
```

Comment:
as i understood it is gate for dark mode in app ?

Reply:
yes, but we have Light theme forced so it ill not be applied, but required for RN upgrade